src/arch/x86/config.cmake

```diff
@@ -62,7 +62,7 @@ config_choice(
 )
 
 config_option(
-    KernelUseLogcalIDs USE_LOGCAL_IDS
+    KernelUseLogicalIDs USE_LOGICAL_IDS
```

Comment:
can this feature be queried with CPUID?

Reply:
Could be, but I don't think anything actually uses this.